pom.xml

```diff
@@ -1826,6 +1830,26 @@
       </properties>
     </profile>
 
+    <profile>
+      <!--
+          Use this profile only for making Spark releases. Note that due to SPARK-8819,
+          you must use maven version 3.2.x or before to avoid running into MSHADE-148.
+      -->
+      <id>release-profile</id>
```

Comment:
Can you rename this to just be called `release`? All of the profiles are, well, profiles. As a convention we don't include `profile` in the name.